client/post-editor/editor-sidebar/index.jsx

```diff
@@ -39,37 +45,50 @@ export default class EditorSidebar extends Component {
 			type,
 			setPostDate,
 			isPostPrivate,
+			nestedSidebar,
+			toggleNestedSidebar,
```

Comment:
In a world where EditorSidebar is more generic and reusable, I would be more careful. In the current world where PostEditor is the ground truth and provides those defaults, I feel like it would be introducing unnecessary "defensive-programming-like" code?

In other words, if we were to chase more reusability for the `EditorSidebar`, I wouldn't chase it in the default props first ;) 

Now if you want I can squeeze a `NESTED_SIDEBAR_NONE` / `noop` in there, as the component (and its children) will break if toggleNestedSidebar isn't a defined function?